src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,160 @@
+package duke;
```

Comment:
The `Parser` could be refactored into its own package using IntelliJ. Similar for `Storage` and `Ui`.
```suggestion
package duke.parser;
```